service/gcs/core/gcs/cleanup.go

```diff
@@ -47,11 +47,18 @@ func (c *gcsCore) cleanupContainer(containerEntry *containerCacheEntry) error {
 			errToReturn = err
 		}
 	}
-	if err := c.destroyContainerStorage(containerEntry.ID); err != nil {
-		logrus.Warn(err)
-		if errToReturn == nil {
-			errToReturn = err
+
+	// We only do cleanup
```

Comment:
Is this supposed to say something like "We only do cleanup if unmounting succeeds"?

Reply:
Lol. Guess I got sidetracked. Fixed.